package.json

```diff
@@ -21,13 +21,14 @@
     "homepage": "https://github.com/Exabyte-io/materials-designer",
     "dependencies": {
         "@babel/cli": "7.0.0",
-        "@babel/core": "7.0.0",
+        "@babel/core": "^7.1.0",
```

Comment:
updates from https://github.com/Exabyte-io/materials-designer/pull/28/files

Reply:
If we are updating package versions we need to update package-lock too. It would be better to have two PRs under this tasks's identifies then:
(1) where we merge the changes from https://github.com/Exabyte-io/materials-designer/pull/28/files into the epic branch you are starting from - `feature/SOF-5098-update-packages`
(2) where we introduce the remaining logic `feature/SOF-5098` into `feature/SOF-5098-update-packages`